docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
+
+The types and features of each benefit can vary between payroll systems, and even between users of the same payroll system, depending on the configuration that the company has set up. For this reason, we provide a `/employer/benefits/meta` endpoint, which will provide the types and features available for the employer whose benefits you are managing.
 
 On request, we can also provide written documentation about supported features and benefit types at the provider level. Please reach out to your account representative for that information.
 
-## Enrolling and Unenrolling Individuals
+**Note: One-time post-tax deductions are not currently supported for benefits yet.**
+
+### Handling Assisted Benefits API responses
+
+For Automated connections that have Assisted Benefits enabled, the response you receive from the `/employer/benefits` endpoints will contain a `job_id`. While our dedicated product operations team makes the necessary changes in the provider's system, you can continue to call the `GET /jobs/{job_id}` to get the status of the job. The valid job status responses will be either `pending`, `in_progress`, `complete`, or `error` with a response body further explaining the response.
+
+Example `POST /employer/benefits` response:
+
+```json
+{
+  "benefit_id": "264122c0-0216-4f21-a4cd-7b3bcddbe3fc",
+  "job_id": "497d98f3-580a-4ab9-830a-af2346d029b2"
+}
+```
+
+Example `POST /employer/benefits/{benefit_id}/individuals` Response:
+
+```json
+{
+  "job_id": "3a82a144-d168-4207-942a-a5852b11df1c"
+}
+```
+
+This will be the same for `DELETE /employer/benefits/{benefit_id}/individuals`
+
+Example `GET /jobs/{job_id}` response after initial job submission:
+
+```json
+{
+  "job_id": "3a82a144-d168-4207-942a-a5852b11df1c",
+  "code": 202,
+	"status": "pending"
+}
+```
+
+Example `GET /jobs/{job_id}` response after job is completed for a `POST /employer/benefits/individuals` request with multiple individuals:
+
+```json
+{
+ "job_id": "3a82a144-d168-4207-942a-a5852b11df1c",
+  "code": 207,
+  "status": "complete",
+ "body": [
+    {
+      "individual_id": "430f9d95-1dcf-4b99-b616-45f814416890",
+      "code": 201,
+      "message": "Successfully enrolled individual in benefit",
+    },
+    {
+      "individual_id": "647975ac-1e0f-4e9c-b705-e3042da48581",
+      "code": 200,
+      "message": "Successfully updated enrollment for individual",
+    },
+    {
+      "individual_id": "4a0a3b15-d3d6-41c2-a4a4-ca4ed1b68cf8",
+      "code": 404,
+      "message": "Individual not found"
+    },
+    ...
+  ]
+}
+```
+
+## Gusto Compatibility Matrices
```

Comment:
did we get the ok to include this publicly again? if so, our compatibility section might be a better place for it. this guide is quite long already

Reply:
@alexromano Not sure, I can't get a straight answer on why it was hidden in the first place. The most I could figure out was, "it was hidden early on as we were building it, but as it is more widely used product, most of this stuff should be public." 

I am not sure how making this data public could hurt us? All of this data is available via the /benefits/meta endpoint, right?

Reply:
Yeah, its available through the meta endpoint. Only developers with a gusto token would be able to see that though. that being said, I'm ok making it public